src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/MonitoredTripLocks.java

```diff
@@ -17,27 +17,27 @@ public class MonitoredTripLocks {
     /** the amount of time in milliseconds to wait to check if a lock has been released */
     private static final int LOCK_CHECK_WAIT_MILLIS = 500;
 
-    private static final ConcurrentHashMap<MonitoredTrip, Boolean> locks = new ConcurrentHashMap();
+    private static final ConcurrentHashMap<String, Boolean> locks = new ConcurrentHashMap();
```

Comment:
SonarLint: Update to `new ConcurrentHashMap<>()`

Reply:
Refactored in https://github.com/ibi-group/otp-middleware/pull/199/commits/ca89702fd4441423f3af8a721054834fe7bc7d64.